### PR TITLE
:sparkles: Added "#touch-temp" hash flag

### PR DESCRIPTION
**Pull request by EzeKoren**
This flag boots into TouchUI without toggling the flag in localstorage

### CODE REVIEW

**Comment by EzeKoren** [comment]
I believe this very simple feature is very useful for those who use browsers with side panels, like Vivaldi. That way you can have a TouchUI tab and a regular Octoprint tab in the same browser

File: octoprint_touchui/static/js/touchui.bundled.js [hunk 1]
```diff
@@ -691,8 +691,12 @@ TouchUI.prototype.core.init = function() {
...
 		this.isActive(true);
 
-		// Enforce active cookie
-		this.DOM.storage.set("active", true);
+		// Do not set active cookie if hash "#touch-temp" is used
+		// That way you can boot into TouchUI for a single session
+		if (document.location.hash !== "#touch-temp") {
+			// Enforce active cookie 
+			this.DOM.storage.set("active", true);
+		}
```
Comment (EzeKoren) [inline octoprint_touchui/static/js/touchui.bundled.js:699]: For some reason, GitHub displays line changes that are identical to the source. In order to make it legible, this is one of the two changes I'm suggesting.

File: octoprint_touchui/static/js/touchui.bundled.js [hunk 2]
```diff
@@ -741,6 +745,7 @@ TouchUI.prototype.core.boot = function() {
 	// This should always start TouchUI
 	if(
 		document.location.hash === "#touch" ||
+		document.location.hash === "#touch-temp" ||
```
Comment (EzeKoren) [inline octoprint_touchui/static/js/touchui.bundled.js:748]: And this is the second change

**Comment by BillyBlaze** [comment]
Hi @EzeKoren, thanks for the pull request. I have no problem merging this, however `touchui.bundled.js` is a generated file. So your changes will disappear. Can you do your changes in the following files:

- https://github.com/BillyBlaze/OctoPrint-TouchUI/blob/master/source/js/core/boot.js
- https://github.com/BillyBlaze/OctoPrint-TouchUI/blob/master/source/js/core/_init.js#L25

**Comment by EzeKoren** [comment]
Absolutely! Thank you for considering this change and I look forward to seeing it on a future release

**Comment by EzeKoren** [comment]
And done! Would you prefer that I rollback the changes I did to the bundled file so you can run the compiler?

**Comment by BillyBlaze** [comment]
Thank you for your contribution.

And nah, the changes in bundled are ok, this way anyone who installs the maintenance branch will see your changes.

**Pull request merged**